client/js/views/templates/WhereToFindUs.js

```diff
@@ -1,3 +1,25 @@
 module.exports = function( { model } ) {
-    return `<div>where to find us</div>`
+	const markets = model.marketData.reduce((memo, market) => {
+		const markup = `<div>
+		<h3>${ market.name }</h3>
+		<p>${ market.address1 }</p>
```

Comment:
I don't think `<p>` is the right element here. I think those should be reserved for sentences/paragraphs. You also end up having to override the margins that the browser sets on `<p>`. Just use a generic `<div>` instead.